requirements/constraints.txt

```diff
@@ -19,10 +19,6 @@
 # Issue for unpinning: https://github.com/openedx/edx-platform/issues/35280
 celery>=5.2.2,<6.0.0
 
-# Date: 2024-02-02
-# Stay on LTS version, remove once this is added to common constraint
-Django<5.0
```

Comment:
Should this be updated to `Django<6.0` rather than being removed?

Reply:
Updated the constraint to `Django<6.0` instead.